fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedFieldInfosFormatTest.java

```diff
@@ -241,7 +244,7 @@ void fewerFieldsInNextSegment2(boolean oneTransaction) throws Exception {
     void mixedFieldNumbers(boolean oneTransaction) throws Exception {
         testWithTwoSegments(oneTransaction,
                 singleFieldInfos("foo", 0),
-                singleFieldInfos("foo", 1));
+                singleFieldInfos("foo", 1), 1);
```

Comment:
This is somewhat concerning to me, I feel like this should result in two separate FieldInfos.